orchagent/fdborch.cpp

```diff
@@ -73,11 +74,12 @@ bool FdbOrch::storeFdbEntryState(const FdbUpdate& update)
 
     if (update.add)
     {
+        entry.port_name = portName;
```

Comment:
Some background: storeFdbEntryState is called in 3 events, namely, LEARN, AGED/MOVE, and FLUSH. All these three calls happen from update() method, which is invoked when we receive notification.

In case of LEARN, we will definitely receive valid bridge_port_id, hence we can derive port object and it will be present in update structure. Now, we can use the port name in update.port.m_alias and fill it in FDB entry structure, before storing in m_entries of FDB class.

In case of AGE/MOVE we will receive a valid bridge_port_id, but its un-necessary as we are going to remove FDB entry.

In case of FLUSH, we may or may NOT receive a valid bridge_port_id, but again it is not necessary for us to know the port name as we are going to delete FDB entry.

Hence I thought, getting the port name from update.port.m_alias and adding it to FDB entry should happen only in LEARN case, which is a update.add == true. But let me know Port and portName also inside that if block.